.gitignore

```diff
@@ -2,3 +2,5 @@
 node_modules
 phpunit.xml
 vendor
+.idea
+composer.lock
```

Comment:
Please revert this change, the `composer.lock` file is always committed so that all installations have the same installed dependencies.